core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -57,7 +56,7 @@ class MessageTest extends JUnitSuite {
   }
 
   @Test
-  def testFieldValues {
+  def testFieldValues = {
```

Comment:
What's the rationale of adding a `=` here? Or should it better to be `def XX(): Unit = {`?